packages/analytics/src/integrations/gtag/GTag.ts

```diff
@@ -48,4 +47,16 @@ export class GTag implements IIntegration {
       this.reset();
     });
   };
+
+  revenue = (price, quantity, type, productId = '1', currency = 'USD') => {
```

Comment:
This doesn't seem to match the `Revenue` type defined in `packages/analytics/src/types.ts`. Should we use the type as we do in the other methods?

Reply:
it does not match?? I see the same type 
`export type Revenue = (
  price: number,
  quantity: number,
  type: string,
  productId?: string | number,
  currency?: string
) => void;`

Reply:
Forget my previous comment, I was checking an outdated version 🙏🏻